File: web-be/src/main/java/com/project/web_be/dtos/SubmissionExamDTO.java
package com.project.web_be.dtos;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class SubmissionExamDTO {
    @JsonProperty("exam_id")
    private Long examId;

    @JsonProperty("student_id")
    private Long studentId;

    @JsonProperty("answers")
    private List<SubmissionAnswerDTO> answers;
} 

File: web-fe/src/app/dtos/requests/submission-exam.dto.ts
import { SubmissionAnswerDTO } from "./submission-answer.dto";


export class SubmissionExamDTO {
    exam_id!: number;
    student_id!: number;
    answers!: SubmissionAnswerDTO[];

    constructor(data: any) {
        this.exam_id = data.exam_id;
        this.student_id = data.student_id;
        this.answers = data.answers;
    }
}
